### PR TITLE
Updated homepage and poll viewer to show teams dropped from rankings

### PR DESCRIPTION
**Pull request by sprx97**

### CODE REVIEW

File: poll/views.py [hunk 1]
```diff
@@ -23,11 +23,25 @@ def home(request):
     up_movers = ranks.order_by('-ppv_diff')[0:5]
     down_movers = ranks.order_by('ppv_diff')[0:5]
 
+    dropped = []
```
Comment (sprx97) [inline poll/views.py:26]: This code is basically duplicated in poll_viewer() and home(). Would you rather me pull it out into a helper function?

File: poll/views.py [hunk 2]
```diff
@@ -23,11 +23,25 @@ def home(request):
...
+    prev_poll = Poll.objects.get(pk=(int(most_recent_poll.pk)-1))
+    if prev_poll.year == most_recent_poll.year: # don't display dropped teams for first poll of year
+        prev_top25 = PollCompare.objects.filter(poll=prev_poll).order_by('rank')[0:25]
+
+        teams = []
```
Comment (sprx97) [inline poll/views.py:31]: I couldn't find a prettier way to do this. Is there a way to say "if team.team.pk not in top25.teams"? I'd have to compare the "rank.team" objects within top25 and prev_top25.

Comment (mrhwick) [reply poll/views.py:31]: An option, but I don't know if it is prettier
```python
dropped_primary_keys = set(prev_top25.values_list("team__pk", flat=True)) - set(top25.values_list("team__pk", flat=True))
dropped = prev_top25.filter(team__pk__in=dropped_primary_keys)
```

Comment (sprx97) [reply poll/views.py:31]: Hmmm I kinda like that better.

File: poll/views.py (hunk 1 above)
Comment (sirgippy) [reply poll/views.py:26]: I think I like the idea of having them be different views. The version in poll_viewer has slightly different information than the one on home by design. There's enough overlap that I could see some of it being brought into a helper function, though.

**Review by sirgippy — APPROVED** [review]

**Pull request merged**